newIDE/app/src/Utils/GDevelopServices/User.js

```diff
@@ -56,3 +57,8 @@ export const getUserPublicProfile = (
     .get(`${GDevelopUserApi.baseUrl}/user-public-profile/${id}`)
     .then(response => response.data);
 };
+
+export const getUserPublicProfileUrl = (userId: string, username: ?string) =>
+  username
+    ? `https://liluo.io/${username}${isDev ? '?dev=true' : ''}`
```

Comment:
to follow the same logic as the other calls, maybe move this logic to ApiConfigs.js inside `GDevelopGamesPlatform`?